Khan/perseus
# [interactive graph editor] Refactor sections

## Summary:
This puts the correct answer and locked figures into their own section so that its easier to get the right things on the screen.

Uploading editor-2.mov…

<img width="832" alt="Screenshot 2024-07-16 at 1 10 43 PM" src="https://github.com/user-attachments/assets/294d5440-9abf-4647-bec1-2c5f7bf7e8e9">



Issue: LEMS-2179

## Test plan:
- Open Storybook
- Open a graph editor
- **You should be able to collapse and expand sections for locked figures and the correct answer**
